etl/steps/export/explorers/minerals/latest/minerals.py

```diff
@@ -180,4 +180,4 @@ def run(dest_dir: str) -> None:
     #
     # Create a new explorers dataset and tsv file.
     ds_explorer = create_explorer(dest_dir=dest_dir, config=config, df_graphers=df_graphers)
-    ds_explorer.save()
+    ds_explorer.write()
```

Comment:
Maybe, instead, it's better to rename `Explorer.write` -> `Explorer.save` for consistency. Up to you!

Reply:
Makes sense. I was overly creative.